packages/google_maps_flutter/google_maps_flutter_web/lib/src/convert.dart

```diff
@@ -243,16 +243,17 @@ gmaps.Size? _gmSizeFromIconConfig(List<Object?> iconConfig, int sizeIndex) {
     final List<Object?>? rawIconSize = iconConfig[sizeIndex] as List<Object?>?;
     if (rawIconSize != null) {
       size = gmaps.Size(
-        rawIconSize[0] as num?,
-        rawIconSize[1] as num?,
+        rawIconSize[0]! as double,
+        rawIconSize[1]! as double,
       );
     }
   }
   return size;
 }
 
 // Converts a [BitmapDescriptor] into a [gmaps.Icon] that can be used in Markers.
-gmaps.Icon? _gmIconFromBitmapDescriptor(BitmapDescriptor bitmapDescriptor) {
+Future<gmaps.Icon?> _gmIconFromBitmapDescriptor(
+    BitmapDescriptor bitmapDescriptor) async {
```

Comment:
Does this method need to be `async`? I don't see any `await`s in it? This should allow this change to simplify a bunch of async/awaits in the rest of the code, I think?